lib/auth/token/verify_token.dart

```diff
@@ -5,8 +5,11 @@ import 'package:http/http.dart' as http;
 import 'jwt.dart';
 
 /// Verify a Firebase token
-void verifyToken(String token, String projectId) => (request) async =>
-    Jwt(token).validate(projectId, await _googleCertificates);
+void verifyToken(String token, String projectId,
```

Comment:
Should return `Future<void>` since it's async (I know I already had it wrong before)

Reply:
@cachapa Is it okay if I replace all the `throw` exceptions with a `return false` instead?

Reply:
I prefer to leave the exceptions in, since they tell the developer what exactly failed, in case they want to log or inform the user.
It should be trivial to wrap it and return false.

Reply:
We're verifying if the token is good. If it's expirary is in the past, then that's a revoke. We shouldn't throw an exception.

Reply:
If the token's expiry is in the past then the token is considered expired, not revoked.

Revoked tokens are when the token has been explicitly made invalid, e.g. the user removes one of his connected devices remotely, or you want to remove access to a user when his subscription ends.
Honestly I'm not even sure how to revoke individual tokens in Firebase Auth. There's an option to disable the user, which probably does the same thing but for all their tokens.

The token can be invalid for a large number of reasons (as you see in the code) and it may be valuable to the developer to know why they're failing. That's why I went through the effort of creating all of those error exceptions.

Reply:
I added `disabled` to the User object, you can see it in the last commit :) That's from firebase_admin.

Anyways, you're right and that's how the java sdk does it, by throwing an exception. In that case, we don't need to return Future<bool>, just Future<void>, yea?

Reply:
github reformated it..
```
we don't need to return Future<bool>, just Future<void>, yea?
```

Reply:
Exactly